### ISSUE TITLE
getReader() is not available.

### ISSUE THREAD

**Issue by ikreymer** [RangerMauve/js-ipfs-fetch#3]
To be compatible with browser fetch (related to #1) and ReadableStream, it should be possible to do:

```
const response = await fetch("ipfs://...")
const reader = response.body.getReader();
```

Currently, getReader() is not a function.

**Comment by RangerMauve** [comment]
Yeah, I think we could integrate WHATWG streams there.

**Comment by RangerMauve** [comment]
Thank you for your patience, fixed in 1.3.0. Should be an actual readable stream now.
